src/runtime/auto-prompt-manager.js

```diff
@@ -187,6 +189,14 @@ export class AutoPromptManager {
     dismissedPrompts,
     params
   ) {
+    const audienceActionsPromptType =
+      article && article.audienceActions && article.audienceActions.actions
+        ? this.getAutoPromptType_(article.audienceActions.actions)
+        : undefined;
+
+    // Override autoPromptType if it is undefined.
+    params.autoPromptType = params.autoPromptType || audienceActionsPromptType;
```

Comment:
With this change, you could remove lines 192-195
```suggestion
    params.autoPromptType = params.autoPromptType || this.getAutoPromptType_(article?.audienceActions?.actions);
```

Reply:
Done.